packages/client/src/client-utils.ts

```diff
@@ -37,11 +37,14 @@ export function encodeCookies(cookies: Record<string, string>) {
   return value.length ? value : undefined;
 }
 
-export function buildHeaders(headers: Record<string, string>, cookies: Record<string, string>) {
+export function buildHeaders(headers: Record<string, string>, cookies: Record<string, string>, bodyType: BodyType) {
   const encodedCookies = encodeCookies(cookies);
   if (encodedCookies) {
     headers['Cookie'] = encodedCookies;
   }
+  if (bodyType) {
```

Comment:
Should we restrict this based on the http method and only allow this for PATCH,PUT and POST ?
Not sure if it would also make sense for DELETE.

Maybe does not make sense to restrict it but it came into my mind

Reply:
I think if we wanted to prevent bodies for methods that shouldn't have them, I'd do that elsewhere, e.g. disallow body inputs using the type system during definition. Not a bad idea, not sure if it's worth spending time on.